soap/permissions.py

```
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+"""
+مجوزهای محلی اپ SOAP
+"""
+
+from typing import Any
```

Ruff (0.12.2)

Module level import not at top of file

(E402)